python/deltalake/schema.py

```diff
@@ -0,0 +1,166 @@
+import json
+from typing import Dict, List, Any
+from enum import Enum
+
+
+class DeltaTableSchemaFormat(Enum):
+    DELTA = "DELTA"
+    ARROW = "ARROW"
+
+
+class DeltaTableField:
+    def __init__(
+        self,
+        name: str,
+        type: str,
+        nullable: bool,
+        metadata: Dict[str, str],
+        format: DeltaTableSchemaFormat,
+    ):
+        type = json.loads(type)
+        if format == DeltaTableSchemaFormat.ARROW:
+            self.type = DataType.from_arrow_json(json_dict=type)
+        else:
+            self.type = DataType.from_delta_json(json_dict=type)
+        self.name = name
+        self.nullable = nullable
+        self.metadata = metadata
+
+    def __str__(self):
+        return f'{self.name}: {self.type} {self.nullable} {self.metadata if self.metadata else ""}'
+
+
+class DeltaTableSchema:
+    def __init__(self, fields: List[DeltaTableField]):
+        self.fields = fields
+
+    def __str__(self):
+        for field in self.fields:
+            print(field)
+        return ""
```

Comment:
`__str__` method needs to return the representation as a string, not print it out in stdout :)

Reply:
Yes, forgot about that one :)